v2/apierror/apierror.go

```diff
@@ -60,6 +61,30 @@ type ErrDetails struct {
 	Unknown []interface{}
 }
 
+// ErrMessageNotFound is used to signal ExtractMessage found no matching messages.
```

Comment:
s/ExtractMessage/ExtractProtoMessage/ (3x in these docs)

Reply:
Done.